chutney/server/src/test/java/com/chutneytesting/execution/infra/storage/DatabaseExecutionHistoryRepositoryTest.java

```diff
@@ -463,6 +464,71 @@ void deletes_executions_by_ids() {
             ).isInstanceOf(ReportNotFoundException.class));
         }
 
+        @Test
+        void campaign_execution_must_be_deleted_when_last_scenario_execution_is_deleted() {
```

Comment:
Could we have the rule explicitly written down ?
Maybe with a NestedClass and appropriate DisplayName ?